lib/forcegraph.js

```diff
@@ -1,744 +1,200 @@
-define(['d3', 'helper'], function (d3, helper) {
+define(['d3-selection', 'd3-force', 'd3-zoom', 'd3-drag', 'forcegraph/math', 'forcegraph/draw'], function (d3Selection, d3Force, d3Zoom, d3Drag, math, draw) {
   'use strict';
 
-  var margin = 200;
-  var NODE_RADIUS = 15;
-  var LINE_RADIUS = 12;
-
   return function (config, linkScale, sidebar, router) {
     var self = this;
+    var el;
```

Comment:
i prefer let and const, but if we rewrite complete file, we should be consistent.